cpp/test/Slice/errorDetection/IdentAsKeyword.ice

```diff
@@ -32,8 +32,8 @@ module Test
     interface object { void op(); }
     interface long { void op(); }
 
-    sequence<long> impLEments;
-    sequence<long> implements;
```

Comment:
This test is named `IdentAsKeyword`, and we're testing that sequence identifiers are correctly checked for keywords.
This only works if the identifier is actually a keyword. So I switched to `extends`.